netaddr/eui/__init__.py

```diff
@@ -100,6 +100,10 @@ def __init__(self, oui):
         else:
             raise NotRegisteredError('OUI %r not registered!' % (oui,))
 
+    def __hash__(self):
+        """:return: hash of this OUI object suitable for dict keys, sets etc"""
```

Comment:
Let's drop this comment, the meaning of the value returned by `__hash__()` is rather standard.

Reply:
Will do.  That was a copy-and-paste from the `EUI.__hash__`.